mlflow/genai/labeling/__init__.py

```diff
@@ -63,21 +63,29 @@ def get_labeling_sessions() -> list[LabelingSession]:
     return get_review_app().get_labeling_sessions()
 
 
-def get_labeling_session(name: str) -> LabelingSession:
+def get_labeling_session(
+    name: Optional[str] = None, run_id: Optional[str] = None
```

Comment:
```suggestion
    *, name: Optional[str] = None, run_id: Optional[str] = None
```

Shall we prevent positional argument? The error might be confusing when users do `get_labeling_session("runs://...")`.

Reply:
Do you think this is still needed now that we only take run id?

Reply:
nope!